src/components/Table/Table.stories.tsx

```diff
@@ -158,3 +158,26 @@ export const FixedWidth: Story = {
     ]
   }
 };
+
+const maxUidTestRows = [...Array.from({ length: 20 }).keys()].map(
```

Comment:
I'm still curious why even 20 rows are needed? The character count of every row is exactly the same. I'd even argue that having a single row of demo data would be easier for users to adjust props on (especially since changing the prop configuration requires editing the array directly).